SoftLayer/CLI/hardware/monitoring.py

```diff
@@ -0,0 +1,37 @@
+"""Get monitoring for a hardware device."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Get details for a hardware monitors device."""
+
+    hardware = SoftLayer.HardwareManager(env.client)
+
+    table = formatting.KeyValueTable(['name', 'value'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+
+    monitoring = hardware.get_hardware(identifier)
+
+    table.add_row(['domain', monitoring.get('fullyQualifiedDomainName')])
+    table.add_row(['public Ip', monitoring.get('primaryIpAddress')])
+    table.add_row(['private Ip', monitoring.get('primaryBackendIpAddress')])
+    table.add_row(['location', monitoring['datacenter']['longName']])
+
+    monitoring_table = formatting.Table(['Id', 'ipAddress', 'status', 'type', 'notify'])
```

Comment:
Capitalize the first letter for column/key names, and "IP" should have both capitalized (at least when its a title/key) since its an acronym 
```suggestion
    table.add_row(['Domain', monitoring.get('fullyQualifiedDomainName')])
    table.add_row(['Public IP', monitoring.get('primaryIpAddress')])
    table.add_row(['Private IP', monitoring.get('primaryBackendIpAddress')])
    table.add_row(['Location', monitoring['datacenter']['longName']])

    monitoring_table = formatting.Table(['Id', 'IP Address', 'Status', 'Type', 'Notify'])
```